packages/client/src/utils/ShoppingCartContext/ShoppingCartContext.jsx

```diff
@@ -16,9 +16,14 @@ export const useShoppingCartContext = () => {
   return useContext(ShoppingCartContext);
 };
 
+const countAllProducts = (products) => {
+  return products.reduce((acc, product) => acc + +product.quantity, 0);
```

Comment:
`acc + +product.quantity` looks a bit weird. Do we need the extra plus?

Reply:
Not realy now. Before this PR `product.quantity` was `typeof String`, that's why I made that. But not we don't need that.